kebechet/config.py

```diff
@@ -36,6 +36,11 @@
 class _Config:
     """Library-wide configuration."""
 
+    # Add more custom deployments to here, and then to services.py.
+    _CUSTOM_DEPLOYMENTS = {
+        "gitlab.cee.redhat.com": "gitlabcee"
```

Comment:
Why is this necessary here? Can we supply this in the configuration file so we don't need to rebuild the application each time we add a deployment?

Reply:
Hm... Ideally, we should probably determine the host from payload - maybe that's a next step, but for later...

Reply:
> Why is this necessary here? Can we supply this in the configuration file so we don't need to rebuild the application each time we add a deployment?

Most likely there are other things that need to be tinkered so I didn't put it in the configuration thus leading to rebuilding, maybe if you want, I can move it to a different file to make it cleaner as we add more custom deployments.